include/render/pixel_format.h

```diff
@@ -0,0 +1,24 @@
+#ifndef RENDER_PIXEL_FORMAT_H
+#define RENDER_PIXEL_FORMAT_H
+
+#include <wayland-server-protocol.h>
+
+struct wlr_pixel_format_info {
+	uint32_t drm_format;
+
+	/* Equivalent of the format if it has an alpha channel, 0 if NA */
```

Comment:
I'd prefer to document that it's `DRM_FORMAT_INVALID` if N/A (which happens to be zero).

Reply:
That's a good idea!